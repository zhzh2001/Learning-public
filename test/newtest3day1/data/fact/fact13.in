7052565 6