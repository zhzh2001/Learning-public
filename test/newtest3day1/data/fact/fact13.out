6.43195e+45235526
